src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsMobileWrapperTest.java

```diff
@@ -88,9 +88,19 @@ private static void iterateNode(Node node, String fileName, String sectionName)
                 Assert.assertTrue(wrapperMessage,
                         node.getAttributes().getNamedItem("class").getNodeValue()
                                 .contains("wrapper"));
+
                 if ("table".equals(child.getNodeName())) {
                     iterateNode(child, fileName, sectionName);
                 }
+                if ("img".equals(child.getNodeName())) {
+                    final String dataImageInlineMessage = fileName + "/" + sectionName + ": img "
+                            + "needs an attribute data-image-inline which has the value 'true' if "
+                            + "the image should be displayed in the same line as the surrounding "
+                            + "text.";
```

Comment:
I don't understand the message versus the actual check.

Shouldn't the message say something like "node needs block or inline class attribute"?

Reply:
Yeah, I will update this accordingly.

Reply:
This looks done and is better.